ui/js/values.js

```diff
@@ -172,6 +172,52 @@ treeherder.value("phAlertResolutionMap", {
     DUPLICATE: 5
 });
 
+treeherder.value("phTalosDocumentationMap", {
```

Comment:
can you add a comment here mentioning what part(s) of this is hacky and indicate that we should revisit this in the future.